stdlib/build.mjs

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+// 2>/dev/null; exec /usr/bin/env node --harmony-top-level-await "$0"
+
+import { promisify } from 'util';
+import { createRequire } from 'module';
+
+import globby from 'globby';
+import replaceExt from 'replace-ext';
+
+// Making asc interop with ESM
+const require = createRequire(import.meta.url);
+const asc = require('assemblyscript/cli/asc');
+
+const compileFile = promisify(asc.main);
+
+// Binaryen is ready
```

Comment:
I feel like this will confuse people since this is AssemblyScript's Binaryen and people might think this has something to do with ours.

```suggestion
// AssemblyScript compiler is ready
```

Reply:
Yeah, makes sense. I literally stole that line from their implementation 😛